wiki/Guides/Compressing_files/en.md

```diff
@@ -58,20 +68,22 @@ This section will show you how to remove audio from videos using [Handbrake](htt
 
 ### Using FFmpeg
 
-This section will show you how you can use [FFmpeg](https://ffmpeg.org/) to reduce video file size. FFmpeg is a program used through a [command-line interface (CLI)](https://en.wikipedia.org/wiki/Command-line_interface), meaning it does not have any [graphical user interface (GUI)](https://en.wikipedia.org/wiki/Graphical_user_interface) by itself. While this may seem intimidating, FFmpeg can offer more flexibility than other tools, such as being able to re-encode multiple video files more quickly and easily.
+**FFmpeg is a program used through a [command-line interface (CLI)](https://en.wikipedia.org/wiki/Command-line_interface), meaning it does not have any [graphical user interface (GUI)](https://en.wikipedia.org/wiki/Graphical_user_interface) by itself.** While this may seem intimidating, FFmpeg can offer more flexibility than other tools, such as when integrated into a script.
+
+To install FFmpeg on Windows, [download FFmpeg](https://ffmpeg.org/download.html) and add its directory to your `PATH` environment variable. On macOS, you can alternatively install it using the [brew](https://brew.sh/) package manager.
 
-If you are on Windows, first [download FFmpeg](https://ffmpeg.org/download.html) and add its directory to your `PATH` environment variable. On MacOS, you can alternatively install it using [brew](https://brew.sh/). Most Linux distributions already have FFmpeg installed, but it can often be easily installed through their respective package managers.
+On Linux, most Linux distributions either already provide or pre-install FFmpeg by default but some may not provide it. Users of such distributions should research their respective distribution for more information.
    
-Open a terminal and paste in the following command, changing the values as needed:
+To use FFmpeg to re-encode a video file, open a terminal and paste in the following command, changing the values as needed:
 
 ```
-ffmpeg -i input -c:v libx264 -crf 20 -preset slower -profile:v high -vf scale=-1:720 -an -sn -map_metadata -1 -map_chapters -1 output.mp4
+ffmpeg -i input -c:v libx264 -crf 20 -preset veryslow -profile:v high -vf scale=-1:720 -an -sn -map_metadata -1 -map_chapters -1 output.mp4
```

Comment:
out of curiosity, does handbrake use ffmpeg under the hood?

Reply:
Handbrake I think doesn't use FFmpeg under the hood but for H.264 (other formats too but this is H.264-specific), it uses the same encoder (open-source x264 encoder) so any options to configure the underlying encoder will work on both.

Reply:
Also, following my comment to setting the Encoder Profile to High on Handbrake that I decided should be removed entirely, on FFmpeg, encoder profile is set by `-profile:v high`, so to follow with Handbrake, it can just be removed or replaced with `-profile:v main` to explicitly use the Main profile.

Differences between those profiles are in what compression technique it uses, but from what I've read and test myself, on low resolutions the benefits of using the High profile is minimum at best.